packages/expo/src/Speech/Speech.types.ts

```diff
@@ -0,0 +1,19 @@
+export type SpeechEventCallback = (this: SpeechSynthesisUtterance, ev: SpeechSynthesisEvent) => any;
+
+export type Options = {
```

Comment:
name this SpeechOptions so if we globally export it, it will have a less plain name and be less likely to conflict with other exports